clang/examples/DPCT/README.md

```diff
@@ -0,0 +1,9 @@
+# How to add API mapping cases
+
+- Add the API mapping cases of different librerays in corresponding folder.
+
+- Make sure the file name is the same as the query API name.
+
```

Comment:
content of API_name.cu, please give an example, and for the example, please highlight the note on each part:
1. function define 
2. Arg list and comments. 
3. put calls between "// start" and "// end", all other code, don't put in this scope.
4. how and when "// option" is required.